R/PIConfiguration.R

```diff
@@ -110,20 +114,41 @@ PIConfiguration <- R6::R6Class(
       private$.simulateSteadyState <- FALSE
       private$.steadyStateTime <- 1000
       private$.printEvaluationFeedback <- FALSE
-      private$.targetFunctionType <- "lsq"
+      private$.objectiveFunctionOptions <- list(
```

Comment:
I hope this is described somewhere :D

Reply:
Description still needed. I guess it would be a good to have it somewhere together with the description of the optimization algorithm options